src/app/components/core/configuration-manager/configuration-group/configuration-group.component.css

```diff
@@ -25,5 +25,78 @@
 }
 
 .tabs {
+  margin-right: auto !important;
+  margin-left: auto !important;
   margin-bottom: 0px !important;
 }
+
+#group_navigation .nav {
+  display: flex;
+  flex-wrap: inherit;
+  margin-bottom: inherit;
+}
+
+/*--- Wrap Up ---*/
+.configuration_group_wrapper {
+  position: relative;
+  box-sizing: border-box;
+}
+
+.groupNav {
+  /* Make this scrollable when needed */
+  overflow-x: auto;
+  /* We don't want vertical scrolling */
+  overflow-y: hidden;
+  /* For WebKit implementations, provide inertia scrolling */
+  -webkit-overflow-scrolling: touch;
+  /* We don't want internal inline elements to wrap */
+  white-space: nowrap;
+  /* If JS present, let's hide the default scrollbar */
+  position: relative;
+  /* font-size: 0; */
+}
+
+.groupNav_Contents {
+  float: left;
+  padding-right: 10%;
+  transition: transform 0.2s ease-in-out;
+}
+
+.groupNav_Contents-no-transition {
+  transition: none;
+}
+
+.Advancer {
```

Comment:
We always use class/id name in lowercase. Is there any specific reason of not using lowercase for these some classes in this css file?